_posts/2020-10-12-redesigning-the-sensenetcom-website.md

```diff
@@ -0,0 +1,51 @@
+---
+
+title: "Redesigning the sensenet.com website "
+author: [CsabaPeter]
+image: "../img/posts/sensenetcom-redesign.png"
+tags: [redesign, ux, research, website, webdesign]
+redirect_to: https://www.sensenet.com/blog/2020-10-12-redesigning-the-sensenetcom-website
+
+---
+
+As lot of you already know we came out with the SNaaS (sensenet as a service) beta for the hacktoberfest and with the whole new admin system it’s only fair that we would do a whole new website for sensenet.com as well from scratch. 
+
+---
+
+In early summer when it became clear that we want to launch the new SNaaS in October we also decided to redesign our website as well. Given that it was our goal to finish the site in September so we can properly promote our product and make a site that gives easy to access information about it. The schedule was very tight but at the end of the day the team made it happened. 
+
+## The research 
+
+This time I didn’t had to do the research alone because Blackholemedia a marketing agency helped us out in the process. They did the personas and competitive research and we had sessions and small workshops together where we made customer journey maps and the sitemap as well. Because of the recent pandemic situation in the beginning we did everything online and used Miro during our sessions where everyone could brainstorm and share his or her ideas at the same time.  
```

Comment:
```suggestion
This time I didn’t had to do the research alone because Blackhole Media (a marketing agency) helped us out in the process. They did the personas and competitive research and we had sessions and small workshops together where we made customer journey maps and the sitemap as well. Because of the recent pandemic situation in the beginning we did everything online and used Miro during our sessions where everyone could brainstorm and share his or her ideas at the same time.  
```